.heroku/vendor/pygobstones-lang

```diff
@@ -1 +1 @@
-Subproject commit 141d657e1317132a215f54cec952580032d86f6d
+Subproject commit 34e62b00bf7e0ce23094be7c9a8cc6237c6ebdde
```

Comment:
![you-are-the-qp259a](https://cloud.githubusercontent.com/assets/1585835/9558188/0e04d854-4db9-11e5-9835-07b1bf23cc57.jpg)